README.md

```diff
@@ -1,34 +1,18 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Atom Linter](https://github.com/AtomLinter/Linter) providing an interface to [HTMLHint](https://github.com/yaniswang/HTMLHint). It will be
 used with files that have the syntax.
 
 ## Installation
 
-Linter package must be installed in order to use this plugin. If Linter is not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
-
-### Plugin installation
-
-```ShellSession
+```bash
```

Comment:
`bash` isn't quite correct, this should be left as `ShellSession`.

----

[bash](https://github.com/github/linguist/blob/v4.7.3/lib/linguist/languages.yml#L3247) is an alias of [Shell](https://github.com/github/linguist/blob/v4.7.3/lib/linguist/languages.yml#L3241), which is inteded for scripting files.

[ShellSession](https://github.com/github/linguist/blob/v4.7.3/lib/linguist/languages.yml#L3267) is meant to be used for documenting console sessions. Since this is intended to be what the user types into their shell, it's the more correct choice here.